AUTOMATION.md

```diff
@@ -0,0 +1,40 @@
+# Automation
+
+There are some Organization-wide secrets available for GitHub Automation of various workflows.
+
+## Personal Access Tokens
+
+Automation currently depends on the availability of two GitHub Personal Access Tokens (PATs).
+
+PATs have an expiry and the secret will have to be updated periodically by admins. An admin will create
+a PAT on their account, requesting access to the organization. See [the docs](https://docs.github.com/en/organizations/managing-programmatic-access-to-your-organization/managing-requests-for-personal-access-tokens-in-your-organization).
+
+### Release PAT
+
+The release PAT requires permissions:
+- Read access to actions variables, metadata, and secrets for all current and future repositories owned by this
+  organization
+- Read and Write access to code, issues, and pull requests for all current and future repositories owned by this
+  organization
+
+### Sonar PAT
+
+The sonar PAT requires permissions:
+- Read access to metadata for all current and future repositories owned by this organization
+- Read and Write access to code and pull requests for all current and future repositories owned by your organization
```

Comment:
The Sonar token is defined/configured in Sonar Cloud itself, rather than being a GitHub PAT.

Reply:
Updated. Does it have an expiry or anything else worth noting?